pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/AdopsjonRevurdering.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.pensjon.etterlatte.maler.barnepensjon
+
+import no.nav.pensjon.brev.template.Language
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+import no.nav.pensjon.etterlatte.EtterlatteBrevKode
+import no.nav.pensjon.etterlatte.EtterlatteTemplate
+import no.nav.pensjon.etterlatte.maler.BarnepensjonRevurderingAdopsjonDTO
+import no.nav.pensjon.etterlatte.maler.BarnepensjonRevurderingAdopsjonDTOSelectors.adoptertAv
+import no.nav.pensjon.etterlatte.maler.BarnepensjonRevurderingAdopsjonDTOSelectors.virkningsdato
+import no.nav.pensjon.etterlatte.maler.NavnSelectors.etternavn
+import no.nav.pensjon.etterlatte.maler.NavnSelectors.fornavn
+import no.nav.pensjon.etterlatte.maler.fraser.barnepensjon.Adopsjon
+import no.nav.pensjon.etterlatte.maler.fraser.barnepensjon.Barnepensjon
+
+@TemplateModelHelpers
+object AdopsjonRevurdering : EtterlatteTemplate<BarnepensjonRevurderingAdopsjonDTO> {
+    override val kode: EtterlatteBrevKode = EtterlatteBrevKode.BARNEPENSJON_REVURDERING_ADOPSJON
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = BarnepensjonRevurderingAdopsjonDTO::class,
+        languages = languages(Language.Bokmal),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Vedtak - opphør på grunn av adopsjon",
+            isSensitiv = true,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VEDTAK,
+            brevtype = LetterMetadata.Brevtype.VEDTAKSBREV,
+        ),
+    ) {
+        title {
+            text(
+                Language.Bokmal to "Vi stanser barnepensjonen din",
+            )
+        }
+        outline {
+            includePhrase(
+                Adopsjon.BegrunnelseForVedtaket(
+                    virkningsdato = virkningsdato,
+                    navn = adoptertAv.fornavn + " " + adoptertAv.etternavn,
```

Comment:
Må korrespondere med evt. mellomnavn i ey-repoet -- lurer på om det allerede er typer for navn med mulig mellomnavn for andre etterlatte-brev :)

Reply:
For innvilgelse har vi `String navn` 🙈

Reply:
haha 🤠